playground/main.js

```diff
@@ -85,15 +85,19 @@ const renderPlaygroundWidget = (options = {}) => {
 
     // assert english leaks if locale set to ok-PL
     if (signinWidgetOptions.language === 'ok-PL') {
-      // wait for view to finish rendering
-      const viewText = document.getElementById('okta-sign-in').textContent;
+      //Use innerText to avoid including hidden elements
```

Comment:
This is tricky, as some "hidden" elements may still have HTML properties that need to be read via screen readers. Probably OK to hide this for now, but something to consider when we ramp up our a11y testing.